emva1288/camera/routines.py

```diff
@@ -168,7 +223,9 @@ def get_bayer_filter(t00, t01, t10, t11, width, height):
         The array with the bayer filter of the size gived.
     """
     pattern_rep = np.array([[t00, t01], [t10, t11]])
-    size = (int(np.floor(height / pattern_rep.shape[1]))+1,
-            int(np.floor(width / pattern_rep.shape[0]))+1)
+    size = (height, width)
```

Comment:
because we call np.tile and not get_tile and for np.tile we need to give the size like that.